src/etc/ssl/opnsense.cnf

```diff
@@ -369,3 +369,16 @@ extendedKeyUsage=serverAuth,1.3.6.1.5.5.8.2.2
 keyUsage = digitalSignature, keyEncipherment
 
 ###OPNsense:subjectAltName###
+
+[ combined_server_client ]
+
+basicConstraints=CA:FALSE
+nsCertType			   = server, client
```

Comment:
there was chatter a while back that says we don't need "nsCertType" and "nsComment" anymore.

FWIW, server, client seems wrong while the user said order matters but since it's most likely unused it might be a good start to omit these and go from there?

Reply:
I have looked on the internet and `nsCertType` indeed appears to be unused nowadays. Most likely, `extendedKeyUsage` now takes care of it.

Not sure if dropping `nsComment` is wise - I haven't found an alternative for it, and I still think it could be useful to have a comment.